modules/animation/src/animationcontroller.cpp

```diff
@@ -87,42 +88,18 @@ class Layer;
 
 namespace animation {
 
-namespace {
-
-constexpr std::string_view defaultImageExt = "png";
-
-std::vector<std::string> imageExts(InviwoApplication* app) {
-    return util::transform(util::getDataWriterFactory(app)->getExtensionsForType<Layer>(),
-                           [](const auto& i) -> std::string { return toString(i); });
-}
-
-size_t imageExtIndex(InviwoApplication* app, std::string_view ext) {
-    auto exts = util::getDataWriterFactory(app)->getExtensionsForType<Layer>();
-    auto it = std::find_if(exts.begin(), exts.end(), [&](auto& e) { return e.extension_ == ext; });
-    if (it != exts.end()) {
-        return std::distance(exts.begin(), it);
-    } else {
-        return 0;
-    }
-}
-
-}  // namespace
-
 AnimationController::AnimationController(Animation& animation, InviwoApplication* app)
-    : deltaTime_(Seconds(1.0 / 25.0))
-    , playOptions("PlayOptions", "Play Settings")
+    : playOptions("PlayOptions", "Play Settings")
     , playWindowMode("PlayFirstLastTimeOption", "Time",
                      {{"FullTimeWindow", "Play full animation", 0},
                       {"UserTimeWindow", "Selected time window", 1}},
                      0)
     , playWindow("PlayFirstLastTime", "Window", 0, 10, 0, 1e5, 1, 0.0,
                  InvalidationLevel::InvalidOutput, PropertySemantics::Text)
-    , framesPerSecond(
-          "PlayFramesPerSecond", "Frames per Second",
-          [this]() { return 1.0 / std::abs(deltaTime_.count()); },
-          [this](const double& value) { deltaTime_ = Seconds(1.0 / value); },
-          {0.001, ConstraintBehavior::Immutable}, {1000.0, ConstraintBehavior::Immutable}, 1.0,
-          InvalidationLevel::InvalidOutput, PropertySemantics::Text)
+    , framesPerSecond("PlayFramesPerSecond", "Frames per Second", 25.0,
```

Comment:
Out of scope of PR, but not sure why it was 25 before. I think that 24, 30, or 60 would make most sense.
```suggestion
    , framesPerSecond("PlayFramesPerSecond", "Frames per Second", 24.0,
```